src/components/group-management/leave-group-dialog/index.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+
+import { IconButton, Button } from '@zero-tech/zui/components';
+import { IconXClose } from '@zero-tech/zui/icons';
+
+import './styles.scss';
+
+import { bemClassName } from '../../../lib/bem';
+import { LeaveGroupDialogStatus } from '../../../store/group-management';
+
+const cn = bemClassName('leave-group-dialog');
+
+export interface Properties {
+  name: string;
+  status: LeaveGroupDialogStatus;
+
+  onClose: () => void;
+  onLeave: () => void;
+}
+
+interface State {}
+
+export class LeaveGroupDialog extends React.Component<Properties, State> {
+  render() {
+    return (
+      <div {...cn('')}>
+        <div {...cn('title-bar')}>
+          <h3 {...cn('title')}>Leave Group ?</h3>
+          <IconButton {...cn('close')} size='large' Icon={IconXClose} onClick={this.props.onClose} />
+        </div>
+
+        <div {...cn('body')}>
+          Are you sure you want to leave {this.props.name}? You will lose access to the conversation and its history.
+        </div>
+
+        <div {...cn('footer')}>
+          <Button {...cn('text-button')} variant='text' onPress={this.props.onClose}>
+            <div {...cn('text-button-text')}>Cancel</div>
```

Comment:
I'm not sure why we need an extra `div` here. Button's are supposed to "just work" if we use the right variant, aren't they?

Reply:
I added an extra `div` to update the CSS of the "text". Actually we're supposed to update the zUI buttons to add "size" variant (small, medium, large). But since we don't have that yet, i have added this div to apply the styling for "large text". 

I have added a TODO
<img width="724" alt="image" src="https://github.com/zer0-os/zOS/assets/33264364/6e37f825-fbf4-40e5-a7da-f10f6e525077">